mlir/lib/Dialect/SCF/IR/SCF.cpp

```diff
@@ -3872,14 +3872,89 @@ struct WhileRemoveDuplicatedResults : public OpRewritePattern<WhileOp> {
     return success();
   }
 };
+
+/// If both ranges contain same values return mappping indices from args1 to
```

Comment:
Isn't this actually a mapping from `args2` to `args1`?

Reply:
fixed